src/types/severa/workHour/workHour.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Interface for WorkHoursModel
+ */
+interface WorkHoursModel {
+severaWorkHoursId: string;
+user: WorkHoursUserModel;
+project: WorkHoursProjectModel;
+phase: WorkHoursPhaseModel;
+description: string;
+eventDate: string;
+quantity: number;
+startTime?: string;
+endTime?: string;
```

Comment:
Indentation for all interfaces, the content of the interface should be indented as it is in all the other types files